### PR TITLE
feat(cmd): give a birth to cel-shed and p2p key utilities

### PR DESCRIPTION
**Pull request by Wondertan**
Start of #276

Tested manually.

### CODE REVIEW

File: CHANGELOG-PENDING.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Month, DD, YYYY
 ### BREAKING CHANGES
 
 - [go package] (Link to PR) Description @username
+[feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
```
Comment (liamsi) [inline CHANGELOG-PENDING.md:10]: What is `cel-shed`?

Comment (liamsi) [reply CHANGELOG-PENDING.md:10]: Oh, I see: 
image

**Review by jbowen93 — DISMISSED** [review]
LGMT

File: Makefile [hunk 2]
```diff
@@ -14,12 +14,24 @@ build:
...
 
-## install: Builds and installs the celestia-node binary into the GOBIN directory.
+## install: Build and install the celestia-node binary into the GOBIN directory.
 install:
 	@echo "--> Installing Celestia"
 	@go install -ldflags ${LDFLAGS}  ./cmd/celestia
 .PHONY: install
 
+## shed: Build cel-shed binary.
+shed:
```
Comment (renaynay) [inline Makefile:24]:
```suggestion
build-shed:
```

**Review by renaynay — COMMENTED** [review]
small nit, otherwise lgtm

File: Makefile (hunk 2 above)
Comment (Wondertan) [reply Makefile:24]: well, this is not Makefile idiomatic tbh

Comment (renaynay) [reply Makefile:24]: so why is install-shed? let's make them consistent please.

Comment (Wondertan) [reply Makefile:24]: And the thing I did is similar tbh , it should exactly match the made file name - `cel-shed`

Comment (renaynay) [reply Makefile:24]: actually you're right, let's just leave it make shed.

**Review by renaynay — APPROVED** [review]
u were right.

**Comment by Wondertan** [comment]
I'll change it cel-shed to match Makefile rules

**Review by renaynay — APPROVED** [review]

**Review by jbowen93 — APPROVED** [review]

**Pull request merged**